Bottom: d271705eb3aa3950309213c07dc0c269052d624f
Top:    68f3badb82fe0b5ae6c9289d7eb0c8a6d7313e6b
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-04-10 16:32:56 -0700

Refresh of af_ib

---

diff --git a/src/cma.c b/src/cma.c
index 2b3cd4d..cb60422 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -207,7 +207,6 @@ static void ucma_set_af_ib_support(void)
 	ret = rdma_bind_addr(id, (struct sockaddr *) &sib);
 	af_ib_support = !ret;
 
-printf("af_ib support %d", af_ib_support);
 	rdma_destroy_id(id);
 }
 
@@ -905,7 +904,7 @@ printf("rdma_resolve_addr init dest %s\n", ucma_addr_str(dst_addr));
 
 	ret = ucma_get_ib_route(id_priv, src_addr, dst_addr);
 	if (!ret) {
-		((struct sockaddr_in *) &cmd.dst_addr)->sin_addr.s_addr = htonl(0x7f000001);
+		((struct sockaddr_in *) &cmd.dst_addr)->sin_addr.s_addr = 0;
 		printf("rdma_resolve_addr dest %s\n", ucma_addr_str(&cmd.dst_addr));
 	}
